### PR TITLE
Minor edits for doc on critical rendering path

### PR DESCRIPTION
**Pull request by aegatlin**
I originally intended to remove the phrase "avoid jank" from the introductory section, since it could confuse readers not familiar with the slang, but then also did a full edit-pass on the page.

#### Summary

I made minor edits to the CRP docs

#### Motivation

I think "avoid jank" will not be readily understood by people who aren't familiar with the slang term "jank". It sounds like a technical term, to compound the problem. I also attempted to clean up some grammatical errors, typos, etc., and attempted to improve readability more generally.

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by sideshowbarker** [comment]
Since in the browser-engine/development world, *jank* is in fact a widely-used technical term — and since MDN actually has a glossary entry defining it — at https://developer.mozilla.org/en-US/docs/Glossary/Jank — it seems like another possible (and maybe preferable) change here would be make the word *jank* in *and avoid jank* into a link to https://developer.mozilla.org/en-US/docs/Glossary/Jank

And for the record here, some other places where use of the term *jank* can be found:

- https://developers.google.com/web/fundamentals/performance/rendering
- https://www.html5rocks.com/en/tutorials/speed/rendering/
- https://css-tricks.com/the-fight-against-layout-jank/
- https://www.chromium.org/developers/how-tos/trace-event-profiling-tool/anatomy-of-jank/
- https://addyosmani.com/blog/making-a-site-jank-free/
- https://web.dev/smoothness/
- http://jankfree.org/

**Comment by hamishwillee** [comment]
FWIW I agree with @aegatlin that this word is odd, though its meaning is readily inferred (it was new to me until about 2 months ago). Like the proposal of a glossary entry @sideshowbarker

**Comment by aegatlin** [comment]
Hi @sideshowbarker thank you both for the feedback. I did a full edit in a second commit here before noticing your feedback. While I naively favor avoiding the term unless required, and naively think it is not required here, I do completely respect the feedback of a member, and will be happy to reedit the doc and include a link to `jank`, and put back in all references to it thereafter, if you prefer.

Thank you for the incredibly fast feedback and links! <3

**Comment by aegatlin** [comment]
Also, since I am not part of the browser-development world, it would not bother me whatsoever to learn that my edits are bad and to just close the PR! In trying to help I might not in fact be that helpful 😆

**Comment by aegatlin** [comment]
I can also close this PR myself and open a new one with the link to the jank term and only the minimal other grammar edits. If you'd like.

**Comment by hamishwillee** [comment]
Thanks @aegatlin 


In the first instance it would be good to revert the specific removal of Jank and capture

**Comment by aegatlin** [comment]
Hello @hamishwillee thank you for the feedback. I had time to update my edits in accordance with the feedback of you and @sideshowbarker , thereby reincorporating the jank terminology with a link to the glossary item. Let me know if there's anything else I can do :D

File: files/en-us/web/performance/critical_rendering_path/index.md [hunk 1]
```diff
@@ -24,23 +23,23 @@ A request for a web page or app starts with an HTML request. The server returns
...
 
-CSS has its own set of rules for identifying valid tokens. Remember the C in CSS stands for 'Cascade'. CSS rules cascade down. As the parser converts tokens to nodes, with descendants of nodes inheriting styles. The incremental processing features don't apply to CSS like they do with HTML, because subsequent rules may override previous ones. The CSS object model gets built as the CSS is parsed, but can't be used to build the render tree until it is completely parsed because styles that are going to be overwritten with later parsing should not be rendered to the screen.
+CSS has its own set of rules for identifying valid tokens. Remember the C in CSS stands for 'Cascade'. CSS rules cascade down. As the parser converts tokens to nodes,  descendant nodes will inherit some of the styles of the parent. The incremental processing features don't apply to CSS like they do with HTML, because subsequent rules may override previous ones. The CSS object model gets built as the CSS is parsed, but can't be used to build the render tree until it is completely parsed because styles that are going to be overwritten with later parsing should not be rendered to the screen.
```
Comment (teoli2003) [inline files/en-us/web/performance/critical_rendering_path/index.md:32]: Just a drive-by nit:
```suggestion
CSS has its own set of rules for identifying valid tokens. Remember the C in CSS stands for 'Cascade'. CSS rules cascade down. As the parser converts tokens to nodes, descendant nodes will inherit some of the styles of the parent. The incremental processing features don't apply to CSS like they do with HTML, because subsequent rules may override previous ones. The CSS object model gets built as the CSS is parsed, but can't be used to build the render tree until it is completely parsed because styles that are going to be overwritten with later parsing should not be rendered to the screen.
```

**Review by sideshowbarker — APPROVED** [review]
Looks great to me, but will hold off to merge pending @hamishwillee’s review

File: files/en-us/web/performance/critical_rendering_path/index.md [hunk 2]
```diff
@@ -7,12 +7,11 @@ tags:
   - Reference
 ---
 
-
 The Critical Rendering Path is the sequence of steps the browser goes through to convert the HTML, CSS, and JavaScript into pixels on the screen. Optimizing the critical render path improves render performance.The critical rendering path includes the [Document Object Model](/en-US/docs/Web/API/Document_Object_Model) (DOM), [CSS Object Model](/en-US/docs/Web/API/CSS_Object_Model) (CSSOM), render tree and layout.
```
Comment (hamishwillee) [inline files/en-us/web/performance/critical_rendering_path/index.md:10]:
```suggestion
The Critical Rendering Path is the sequence of steps the browser goes through to convert the HTML, CSS, and JavaScript into pixels on the screen. Optimizing the critical render path improves render performance.
The critical rendering path includes the [Document Object Model](/en-US/docs/Web/API/Document_Object_Model) (DOM), [CSS Object Model](/en-US/docs/Web/API/CSS_Object_Model) (CSSOM), render tree and layout.
```

File: files/en-us/web/performance/critical_rendering_path/index.md [hunk 3]
```diff
@@ -7,12 +7,12 @@ tags:
...
 
 The document object model is created as the HTML is parsed. The HTML may request JavaScript, which may, in turn, alter the DOM. The HTML includes or makes requests for styles, which in turn builds the CSS object model. The browser engine combines the two to create the Render Tree. Layout determines the size and location of everything on the page. Once layout is determined, pixels are painted to the screen.
 
-Optimizing the critical rendering path improves the time to first render. Understanding and optimizing the critical rendering path is important to ensure reflows and repaints can happen at 60 frames per second, to ensure performant user interactions and avoid jank.
+Optimizing the critical rendering path improves the time to first render. Understanding and optimizing the critical rendering path is important to ensure reflows and repaints can happen at 60 frames per second, to ensure performant user interactions, and to avoid [jank](https://developer.mozilla.org/en-US/docs/Glossary/Jank).
```
Comment (hamishwillee) [inline files/en-us/web/performance/critical_rendering_path/index.md:15]: Fix link format
```suggestion
Optimizing the critical rendering path improves the time to first render. Understanding and optimizing the critical rendering path is important to ensure reflows and repaints can happen at 60 frames per second, to ensure performant user interactions, and to avoid [jank](/en-US/docs/Glossary/Jank).
```

File: files/en-us/web/performance/critical_rendering_path/index.md [hunk 4]
```diff
@@ -24,23 +24,23 @@ A request for a web page or app starts with an HTML request. The server returns
 
 DOM construction is incremental. The HTML response turns into tokens which turns into nodes which turn into the DOM Tree. A single DOM node starts with a startTag token and ends with an endTag token. Nodes contain all relevant information about the HTML element. The information is described using tokens. Nodes are connected into a DOM tree based on token hierarchy. If another set of startTag and endTag tokens come between a set of startTag and endTags, you have a node inside a node, which is how we define the hierarchy of the DOM tree.
 
-The greater the number of nodes, the longer the following events in the critical rendering path will take. Measure! A few extra nodes won't make a difference, but divitis can lead to jank.
+The greater the number of nodes, the longer the following events in the critical rendering path will take. Measure! A few extra nodes won't make a big difference, but keep in mind that adding many extra nodes will impact performance, leading to jank.
```
Comment (hamishwillee) [inline files/en-us/web/performance/critical_rendering_path/index.md:27]: Jank in this case is redundant. Removing.
```suggestion
The greater the number of nodes, the longer the following events in the critical rendering path will take. Measure! A few extra nodes won't make a big difference, but keep in mind that adding many extra nodes will impact performance.
```

**Review by hamishwillee — APPROVED** [review]
Yes, it is all much better. Thanks very much @aegatlin !

**Comment by sideshowbarker** [comment]
> Also, since I am not part of the browser-development world, it would not bother me whatsoever to learn that my edits are bad

They’re great edits, not bad ones, and I very much hope the project will get the benefit of future refinements like the ones you made here — made with a lot of care and attention to detail. There are a lot of other MDN that could use improvements like the ones you made here.

> In trying to help I might not in fact be that helpful 😆

You’ve definitely been helpful 🙂

**Comment by hamishwillee** [comment]
What @sideshowbarker said ^^^ . We might nitpick over the details, but this is overall much better.
